README.md

```diff
@@ -12,6 +12,118 @@ dotnet add reference path/to/Styra/OpenApi.csproj
 ```
 <!-- End SDK Installation [installation] -->
 
+## SDK Example Usage (high-level)
+
+All the code examples that follow assume that the high-level SDK module has been imported, and that an `OpaClient` instance was created:
+
+```csharp
+using Styra;
+
+public class MyExample {
+    private string serverURL = "http://opa-host:8181";
+    private string path = "authz/allow";
+    private OpaClient opa;
+
+    public MyExample() {
+        opa = new OPAClient(serverURL);
+    }
+
+    // ...
+}
+```
+
+### Simple query
+
+For a simple boolean response with a dictionary input, use the SDK as follows:
+
+```csharp
+var input = new Dictionary<string, object>() {
+    { "user", "alice" },
+    { "action", "read" },
+};
+
+// (local variable) bool? allowed
+var allowed = opa.check(input, path);
```

Comment:
Yeah, that's an artifact of me borrowing ideas from the Java repo, since the two languages have such similar capabilities. :sweat_smile: I'm not attached to `check`, and I'm just fine switching to something else if that works better across languages/repos.